cmd/oauth/oauthcli/cli.go

```diff
@@ -21,10 +21,11 @@ import (
 type OAuthServiceCliArgs struct {
 	cmd.CommonCliArgs
 	cmd.LoggingCliArgs
-	ServiceAddr     string `arg:"--service-addr, env" default:"0.0.0.0:8000" help:"Service address to listen on"`
-	AllowedOrigins  string `arg:"--allowed-origins, env" default:"https://console.redhat.com,https://console.stage.redhat.com,https://console.dev.redhat.com,https://prod.foo.redhat.com:1337" help:"Comma-separated list of domains allowed for cross-domain requests"`
-	KubeConfig      string `arg:"--kubeconfig, env" default:"" help:""`
-	KubeInsecureTLS bool   `arg:"--kube-insecure-tls, env" default:"false" help:"Whether is allowed or not insecure kubernetes tls connection."`
-	ApiServer       string `arg:"--api-server, env:API_SERVER" default:"" help:"host:port of the Kubernetes API server to use when handling HTTP requests"`
-	ApiServerCAPath string `arg:"--ca-path, env:API_SERVER_CA_PATH" default:"" help:"the path to the CA certificate to use when connecting to the Kubernetes API server"`
+	ServiceAddr      string `arg:"--service-addr, env" default:"0.0.0.0:8000" help:"Service address to listen on"`
+	AllowedOrigins   string `arg:"--allowed-origins, env" default:"https://console.redhat.com,https://console.stage.redhat.com,https://console.dev.redhat.com,https://prod.foo.redhat.com:1337" help:"Comma-separated list of domains allowed for cross-domain requests"`
+	KubeConfig       string `arg:"--kubeconfig, env" default:"" help:""`
+	KubeInsecureTLS  bool   `arg:"--kube-insecure-tls, env" default:"false" help:"Whether is allowed or not insecure kubernetes tls connection."`
+	ApiServer        string `arg:"--api-server, env:API_SERVER" default:"" help:"host:port of the Kubernetes API server to use when handling HTTP requests"`
+	ApiServerCAPath  string `arg:"--ca-path, env:API_SERVER_CA_PATH" default:"" help:"the path to the CA certificate to use when connecting to the Kubernetes API server"`
+	RedirectProxyUrl string `arg:"--redirect-proxy-url, env:REDIRECT_PROXY_URL" default:"" help:"the URL of OAuth redirection proxy used in the tests to maintain predictable callback URL"`
```

Comment:
Is this really only ever to be useful in the tests? Also, I would maybe be more explicit with the name and call it `OAuthRedirectProxyUrl`.